core/src/com/unciv/logic/BackwardCompatibility.kt

```diff
@@ -37,6 +37,19 @@ object BackwardCompatibility {
         removeTechAndPolicies()
     }
 
+    fun GameInfo.migrateGreatGeneralPools() {
+        for (civ in civilizations) civ.greatPeople.run {
```

Comment:
Please add vals for oldName and newName

Reply:
I'm somwewhat confused... the vals are to simplify the data, but they'd only be used in the if statement, no? The assignment still needs the actual members

Reply:
No, I mean
`val oldName = "Great Genreral"
val newName = "Great General"`

Reply:
Oh, I didn't realize I accidentally wrote a typo. There no old/new name, that's just me accidentally typing an extra r

Reply:
All the more reason to make this a val ;)